posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+This view allows to generate access tokens (which are in fact URIs) providing restricted access rights to the storage and limited in time in order to secure the data stored there. Without a signature, a web app will not be able to access the resources of its storage. As far as signature generation is concerned, it is possible to generate a URI with read and write rights on all the storage resources for an unlimited period of time, but be careful about security. It is also possible to provide a read-only access token for one hour. Everything depends on the need.
+
+### Containers
+
+![Figure 20 : the storage containers](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_20.png)
+
+This view lists all the containers created in the storage. It allows to create new ones, to modify them after creation and to delete them. Let's simulate the creation of a new container.
+
+![Figure 21 : creation of a new Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_21.png)
+
+This view is the interface for creating a new container. It is very simple, requiring only the name of the new container and the access level. When you click on the drop-down list, the three options shown in Figure 22 are displayed.
+
+![Figure 22 : editing the access level of a Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_22.png)
+
+The three levels of access to a container are _Private_, _Blob_ and _Container_. Configured on Private, a token is required to access the resources of the container. Configured on Blob, it is possible to access the resources hosted in the storage without an access token. Finally, configured on Container, it is possible to access the contents of the containers without access token.
+
+![Figure 23 : editing the access level of a Container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_23.png)
+
+It is possible to change the accessibility level of a container even after it has been created by right-clicking on it, which can be very useful.
+
+Finally, enter the container to view its contents.
+
+![Figure 24 : content of a Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_24.png)
+
+This view lists all the blobs contained in a container. A container can contain three types of blobs namely _Block blobs_, _Append blobs_ and _Page blobs_.
+
+_Block blobs_ are designed to store text or binary files, and to load large files efficiently. _Append blobs_ are designed for append operations such as logging. Finally, _Page blobs_ are made up of pages of up to 512 bytes that can reach a total size of 8 TB. They are designed for frequent random read/write operations. In the context of our project, we only use _Block blobs_.
```

Comment:
_Block blobs_ are designed to store text or binary files, and to load large files efficiently. _Append blobs_ are designed for append operations such as logging. _Page blobs_ are made up of pages of up to 512 bytes that can reach a total size of 8 TB. They are designed for frequent random read/write operations. In the context of our project, we only use _Block blobs_.